components/style/values/computed/mod.rs

```diff
@@ -39,9 +43,8 @@ pub use self::image::{Gradient, GradientItem, Image, ImageLayer, LineDirection,
 pub use self::gecko::ScrollSnapPoint;
 pub use self::rect::LengthOrNumberRect;
 pub use super::{Auto, Either, None_};
-pub use super::specified::{BorderStyle, UrlOrNone};
+pub use super::specified::{BorderStyle};
```

Comment:
nit: I'm not sure if the lint catches this, but we usually don't wrap in braces single imports.